cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -179,6 +179,8 @@ If you'd like to scale node groups from 0, an `autoscaling:DescribeLaunchConfigu
 
 ## Using AutoScalingGroup MixedInstancesPolicy
 
+> Note: The minimum supported version of MixedInstancePolicy is CA v1.14.x.
```

Comment:
This should read "The minimum version of cluster autoscaler to support MixedInstancePolicy is CA v1.14.x"

The way it currently reads makes it sound like there is a minimum version of AutoScalingGroup.MixedInstancesPolicy, not cluster autoscaler.